client/timeplus/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.3.0"   # noqa: F401
+__version__ = "1.0.0"   # noqa: F401
```

Comment:
are we going to use 1.0.1 in this update?

Reply:
that will be updated with git tag